tha2/util.py

```diff
@@ -137,34 +137,36 @@ def rgba_to_numpy_image(torch_image: Tensor, min_pixel_value=-1.0, max_pixel_val
     return rgba_image
 
 
-def extract_pytorch_image_from_filelike(file, scale=2.0, offset=-1.0):
+def extract_PIL_image_from_filelike(file):
     try:
-        pil_image = PIL.Image.open(file)
+        pil_image = PIL.Image.open(file).convert("RGBA")
```

Comment:
Have you thought about the case where the user opens a file without an alpha channel? 

I believe the alpha channel would always be defaulted to 1, and the output would be ugly even with your zeroing preprocessing. Here's an example: https://imgur.com/6eifxWR.

I don't think it's a good idea to prematurely convert the input image to RGBA. Please reinstate the channel checking code.